sycl/include/CL/sycl/intel/sub_group.hpp

```diff
@@ -119,7 +120,7 @@ struct sub_group {
 
   template <typename T>
   using EnableIfIsArithmetic = detail::enable_if_t<
-    detail::is_arithmetic<T>::value, T>;
+    !is_vec<T>::value && detail::is_arithmetic<T>::value, T>;
```

Comment:
Not sure if Arithmetic should include vector types or not, but new name like `EnableIfIsScalarArithmetic` might help to avoid uncertainty.

Reply:
Renamed.